### ISSUE TITLE
Create branches for SRW App release v1.0

### ISSUE THREAD

**Issue by climbfuji** [ufs-community/ufs-srweather-app#19]
The SRW App release leads decided to cut the release/public-v2 branches before the update to ESMF 8.1.0 bs21 - we want to release code that uses an official release of ESMF.

These are the current hashes for the UFS SRW App, the UFS Weather Model and its submodules, and other libraries/applications.

**ufs-srweather-app**
```
 44324ff19525a370572a9fcd27eb0a3db940444d - Mike or Jeff created branch release/public-v1
```

**ufs-weather-model**
```
fa29a215c5fe882fb837b27a6272b9f72d5cce0e - Jun created branch release/public-v2
```

**Submodules**
```
f61416fef691d9ba39a40df1ce72aa574f54c390 FMS - tagged as 2019.01.03, I think GFDL didn't do any specific branches or tags for the MRW App releases either
6a56b8a0109b41562ba03c689faad93f7d955870 FV3 - Jun created branch release/public-v2
   27cc6ab4e46a3381be77edb1c0fbb4561334b1cc atmos_cubed_sphere - Rusty created branch ufs-release/public-v2 in the NOAA-GFDL repository
   930d96333cd562672a77832fa885805c8ffff0ac ccpp/framework - Dom created branch release/public-v5 (note v5 for CCPP)
   0db75636f26c51b23b97c75c0a515885219bd24d ccpp/physics- Dom created branch release/public-v5 (note v5 for CCPP)
      6ee0b62c1ac6204a89a4e922382b611c16dd5fa7 physics/rte-rrtmgp - Dustin created branch release/public-v5 (note v5 for CCPP stuff)
f6beb4ed8842a099ec59abf17c54502f60d01f84 NEMS - Minsuk created branch release/public-v2
   ca171b95095db4fcd0fc7b01c23d073d90becd99 tests/produtil/NCEPLIBS-pyprodutil - Ratko created branch release/public-v2
a8e2cc1c84a1e6a7deb9eef152edec2992b1a859 stochastic_physics - Phil created branch release/public-v2
```

**Thirdparty libraries, tools, documentation**

```
NCEPLIBS-external 95de67fa71b5088e68ea4154462bd5f346b68b76 - Dom created branch release/public-v2
NCEPLIBS 23a1bcb896cc5ef1eb3c52c86047d217e379ff1a - Dom created branch release/public-v2
ccpp-doc 2bdebbf27255a47cf1ee1b59b98352e5ae6a73ad - Dom created branch release/public-v5
```

Note/question: tags will be created for NCEPLIBS-external, NCEPLIBS and the individual NCEPLIBS-* repositories; branches release/public-v2 for the individual NCEPLIBS-* will only be created when needed?

```
UFS_UTILS (separate from NCEPLIBS for SRW App v1.0 / release/public-v2) - WORKING ON MAJOR UPDATE, THEN CREATE BRANCH
EMC_post (separate from NCEPLIBS for SRW App v1.0) - branch  release/public-v2 was created
```

**Comment by junwang-noaa** [comment]
The ufs-weather-model and fv3 branches release/public-v2 were created.

**Comment by mkavulich** [comment]
@climbfuji We are waiting on the UFS_UTILS release branch to be finalized in order to update the top-level build to use cmake rather than a build script. @kgerheiser has these changes in his fork (https://github.com/kgerheiser/ufs-srweather-app/tree/feature/cmake) but they can't be merged here until the UFS_UTILS PRs are in (https://github.com/NOAA-EMC/UFS_UTILS/pull/118 and Larissa's PR which should be opened shortly) and the appropriate changes for the UFS_UTILS build are incorporated there.

I would recommend waiting for that PR here before creating a release branch.

**Comment by JessicaMeixner-NOAA** [comment]
@climbfuji  is there any objection if the WW3 branch from the 19f3126c832b26ae4c5b27bca6c1dcc676755602 commit is called "release/ufs-public-v2". The reason @Ali.Abdolali and I are thinking to do that is that standalone WW3 users will not get confused between this and the standalone WW3 public releases.

**Comment by climbfuji** [comment]
> @climbfuji is there any objection if the WW3 branch from the 19f3126c832b26ae4c5b27bca6c1dcc676755602 commit is called "release/ufs-public-v2". The reason @Ali.Abdolali and I are thinking to do that is that standalone WW3 users will not get confused between this and the standalone WW3 public releases.

That makes sense to me. We use `release/public-v2` for components of the ufs-weather-model or other repositories where there is no ambiguity. We just need to record it here.

**Comment by climbfuji** [comment]
> @climbfuji is there any objection if the WW3 branch from the 19f3126c832b26ae4c5b27bca6c1dcc676755602 commit is called "release/ufs-public-v2". The reason @Ali.Abdolali and I are thinking to do that is that standalone WW3 users will not get confused between this and the standalone WW3 public releases.

@JessicaMeixner-NOAA we realized that it is best for the SRW App release to simply remove WW3 from ufs-weather-model and remove the cmake option in the top-level `CMakeLists.txt` (and three lines in each of the compile scripts). WW3 is a big folder with many files, unnecessary to carry around, and it is entirely self-contained, therefore it is easy to remove. Sorry for this, but I think you can delete the branch `release/ufs-public-v2`.

**Comment by JessicaMeixner-NOAA** [comment]
The branch release/ufs-public-v2 of WW3 has been deleted.  (FYI: @Ali.Abdolali)

**Comment by climbfuji** [comment]
> The branch release/ufs-public-v2 of WW3 has been deleted. (FYI: @Ali.Abdolali)

Thanks, @JessicaMeixner-NOAA and sorry for the hassle.

**Comment by climbfuji** [comment]
All, I updated the hashes in the description above as of 2020/10/09, except for: UFS_UTILS, EMC_post, workflow, misc.

**Comment by climbfuji** [comment]
@mkavulich can you please let me know if the release branch for the workflow has been created, how it is called, and what the current hash is? Can you also check please if the list above does contain all repositories? I'd like to add the missing information on the workflow and then close the issue. Thanks!

**Comment by mkavulich** [comment]
@climbfuji The regional_workflow branch is created (https://github.com/NOAA-EMC/regional_workflow/tree/release/public-v1), and is already used in Externals.cfg for the release branch. The current hash is https://github.com/NOAA-EMC/regional_workflow/commit/4c08a9dd8faf3c19ed94ce62fcd63b662e6b31b0, but I don't believe it is in it's final form yet; there are a few outstanding changes.

**Comment by climbfuji** [comment]
After the latest merge into the release/public-v2 branch of the ufs-weather-model, I updated all hashes in the description of the issue. Since the code is now frozen (except for bugfixes), I am going to close this issue.
